extensions/wikia/ContentReview/ContentReview.i18n.php

```diff
@@ -59,6 +59,10 @@
 	'content-review-status-in-review' => 'In review',
 	'content-review-status-approved' => 'Approved',
 	'content-review-status-rejected' => 'Rejected,',
+
+	'content-review-feedback-link-text' => 'Provide feedback',
+	'content-review-rejection-explanation' => '==Submitted script change was rejected==
```

Comment:
We could make it a parameter of the `content-review-rejection-explanation` message. But if it's too hacky, we can think of another way to link it correctly.